Gemfile.lock

```diff
@@ -251,4 +251,4 @@ DEPENDENCIES
   jekyll-paginate
 
 BUNDLED WITH
-   1.17.1
+   2.0.2
```

Comment:
It looks like you changed the lockfile (but not the Gemfile) - so either restore this to how it was, or also update the gemflie (this is why the build has an error, the versions are different!)

Reply:
Sorry, I'm not very familiar with ruby or ruby gems.  Are you saying that I should use the same version of bundle or just manually edit the .lock file to match?

Reply:
You can either:

 - edit the Gemfile to match the Lockfile or
 - restore the Gemfile.lock to have the version that was previously there.

Either would work :)

Reply:
Sorry, I'm still confused.  I'm reading up on gemfiles right now; when you say to edit the gemfile to match the lockfile, would it be to require the specific version of `bundle`?

Reply:
Hey don't worry any more about it - I just fixed it for you. When you built the site locally, your version of bundler was different than what is required in the Gemfile. The Gemfile.lock records what was last used/run. So when you didn't update the corresponding Gemfile, it triggered an error in the CI. I'm definitely no expert with ruby or gems either, but the error message is logical that the two need to match. I'm not sure how to deal with having different local deployments (and edits to the Gemfile.lock), but we either could remove the .lock entirely, or just change back like we did here. Let me know if you have other ideas!

Reply:
Thanks for the assist.  The only idea I have right now is maybe having a note added to the `README.md` that discusses how bundler may update the .lock file based on the version that is locally used to test deployment of the page?

Reply:
I actually think we'd do better to remove it and avoid the trouble, see https://github.com/radar/guides/issues/14. 

Thoughts?

Reply:
I agree; I think that makes sense.